dev-libs/hyperscan/hyperscan-5.1.1.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit cmake-utils python-r1
+
+DESCRIPTION="High-performance regular expression matching library"
+SRC_URI="https://github.com/01org/hyperscan/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+HOMEPAGE="https://01.org/hyperscan"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="cpu_flags_x86_ssse3 static-libs"
+
+RDEPEND="${PYTHON_DEPS}
+	dev-util/ragel
+	dev-libs/boost
```

Comment:
Since this is used in DEPEND, does it get linked? Is a subslot binder `:=` needed?

Reply:
All ebuilds are tested on my production system. So the only thing I can say is that it works.

Reply:
I've tested 5.1.1 without python as runtime dependency and it works. My thinking is that It is either not needed at all or needed only at configure or perhaps testing stages

Reply:
I had added Python, because the table in the requirements listed them:

```
In addition, the following software is required for compiling the Hyperscan library:

Dependency | Version | Notes
-- | -- | --
CMake | >=2.8.11 |  
Ragel | 6.9 |  
Python | 2.7 |  
Boost | >=1.57 | Boost headers required
Pcap | >=0.8 | Optional: needed for example code only
```

Not sure, if I had put it to the correct variable RDEPEND. Feel free to change this.

Reply:
It is BDEPEND then.

But it looks like the only place where they actually use it is here:
````cmake
# allow for reproducible builds - python for portability
if (DEFINED ENV{SOURCE_DATE_EPOCH})
      execute_process(
          COMMAND "${PYTHON}" "${CMAKE_MODULE_PATH}/formatdate.py" "$ENV{SOURCE_DATE_EPOCH}"
          OUTPUT_VARIABLE BUILD_DATE
          OUTPUT_STRIP_TRAILING_WHITESPACE)
else ()
    string (TIMESTAMP BUILD_DATE "%Y-%m-%d")
endif ()
message(STATUS "Build date: ${BUILD_DATE}")
```

so It can be avoided with little afford